pages/curriculum/lessons/js0/functions_and_execution_context.mdx

```diff
@@ -65,6 +65,7 @@ Let's break down the above example.
     src="/assets/curriculum/lessons/foundations/params.png"
     width="1000"
     height="400"
+    long='true'
```

Comment:
In case you're wondering why not simply `long`, react prints warnings about boolean passed as props. I'm not sure why. 
```
Warning: Received `true` for a non-boolean attribute `tall`.                       
                                                                                   
If you want to write it to the DOM, pass a string instead: tall="true" or tall={val
ue.toString()}.     
```